torch/csrc/jit/tensorexpr/loopnest.h

```diff
@@ -109,6 +109,17 @@ class TORCH_API LoopNest {
 
   StmtPtr simplify();
 
+  // Sanitize variables and buffer names.
+  // The pass assigns predefined names for loop index variables
+  // (i,j,k,l,m,n,o,p,i1,j1,k1,...) and ensures these names are not conflicting
+  // anywhere. It also removes duplicates from other Buf nad Var names as well
+  // as replaces illegal characters in them with underscores.
```

Comment:
Can you also add a comment to say how the index variables are named uniquely?
Something like `<index_var_name><level / number of index vars>_...`

Reply:
That's mentioned in "(i,j,k,l,m,n,o,p,i1,j1,k1,...)". I didn't want to be overly specific because it's not critical as long as the names are legal and unique.